### PR TITLE
Salvo initial pipeline.

### PR DESCRIPTION
**Pull request by mum4k**
An initial AZP pipeline for the MVP of Salvo. Checks out `nighthawk` and `envoy` repositories, builds and publishes the binaries.

Next step will be to integrate Packer, so that VM images are built.

Pipeline result: https://dev.azure.com/cncf/envoy/_build/results?buildId=131099&view=results

### CODE REVIEW

File: salvo-remote/azure-pipelines/bazel.yml [hunk 1]
```diff
@@ -0,0 +1,73 @@
+parameters:
+- name: ciTarget
+  # The CI target to build, this is the $1 given to ci/do_ci.sh below.
+  displayName: "CI target"
+  type: string
+  default: "build"
+- name: repo
+  # The repository to build in.
+  displayName: "repository"
```
Comment (fei-deng) [inline salvo-remote/azure-pipelines/bazel.yml:9]: Can we keep the first letter to be capital for all the disaplyName?

Comment (fei-deng) [inline salvo-remote/azure-pipelines/bazel.yml:9]: Can we keep the first letter to be capital for all the disaplyName?

**Review by fei-deng — APPROVED** [review]

**Review by gyohuangxin — DISMISSED** [review]
A very great start, lgtm.

File: salvo-remote/azure-pipelines/bazel.yml (hunk 1 above)
Comment (mum4k) [reply salvo-remote/azure-pipelines/bazel.yml:9]: Done, thank you.

**Pull request merged**